atlp-rwanda/vikings-ec-bn
# #184347045 remove expired product

### What does this PR do?
The products should stop listing if the expiry date has passed

### Description of Task to be completed?
Given the expiration date has passed, the product should be removed from
the list of available products
### How should this be manually tested?
1. Checkout to this branch: feature/delist-expired-products-184347045
2. Run the command npm install to install packages
3. Run `npm run dev`
4. Check functionalities of cron job for changing availability of expired products
#### What are the relevant pivotal tracker stories?
[184347045](https://www.pivotaltracker.com/story/show/184347045)
